MyShop/Shop.cs

```diff
@@ -2,30 +2,58 @@
 {
     public class Shop
     {
-        public string shopName;
-        public int openingYear;
-        public int employeeCount;
-        public string city;
+        private string shopName;
+        private int openingYear;
+        private int employeeCount;
+        private string city;
+
+        public string ShopName
+        {
+            get { return shopName; }
+
+            private set { shopName = value; }
+        }
+
+        public int OpeningYear
+        {
+            get { return openingYear; }
+
+            private set { openingYear = value; }
+        }
+
+        public int EmployeeCount
+        {
+            get { return employeeCount; }
+
+            private set { employeeCount = value; }
+        }
+
+        public string City
+        {
+            get { return city; }
+
+            private set { city = value; }
+        }
 
         public Shop(string name, int year)
```

Comment:
Use key word "this" for constructors to avoid duplicate code. Please check other classes

Reply:
Fixed

Reply:
Fixed duplicate